core/src/main/java/net/opentsdb/data/types/numeric/NumericType.java

```diff
@@ -64,21 +64,21 @@
    * @return {@code true} if the value is of integer type (call {@link #longValue()}), 
    * {@code false} if it's of double point type (call {@link #doubleValue()}.
    */
-  public abstract boolean isInteger();
+  public boolean isInteger();
```

Comment:
With TypeTokens do we still need type checks like these?

Reply:
Yeah, for the numeric since I want to use the same interface for int/floats. The type tokens are for the higher level numerics vs annotations.